qa/vagrant/src/test/resources/packaging/scripts/packaging_test_utils.bash

```diff
@@ -305,51 +305,41 @@ clean_before_test() {
                             "/usr/lib/tmpfiles.d/elasticsearch.conf" \
                             "/usr/lib/sysctl.d/elasticsearch.conf")
 
-    if [ "$ES_CLEAN_BEFORE_TEST" = "true" ]; then
-        # Kills all processes of user elasticsearch
-        if id elasticsearch > /dev/null 2>&1; then
-            pkill -u elasticsearch 2>/dev/null || true
-        fi
-
-        # Kills all running Elasticsearch processes
-        ps aux | grep -i "org.elasticsearch.bootstrap.Elasticsearch" | awk {'print $2'} | xargs kill -9 > /dev/null 2>&1 || true
+    # Kills all processes of user elasticsearch
+    if id elasticsearch > /dev/null 2>&1; then
+        pkill -u elasticsearch 2>/dev/null || true
+    fi
 
-        # Removes RPM package
-        if is_rpm; then
-            rpm --quiet -e elasticsearch > /dev/null 2>&1 || true
-        fi
+    # Kills all running Elasticsearch processes
+    ps aux | grep -i "org.elasticsearch.bootstrap.Elasticsearch" | awk {'print $2'} | xargs kill -9 > /dev/null 2>&1 || true
 
-        if [ -x "`which yum 2>/dev/null`" ]; then
-            yum remove -y elasticsearch > /dev/null 2>&1 || true
-        fi
+    # Removes RPM package
+    if is_rpm; then
+        rpm --quiet -e elasticsearch > /dev/null 2>&1 || true
+    fi
 
-        # Removes DEB package
-        if is_dpkg; then
-            dpkg --purge elasticsearch > /dev/null 2>&1 || true
-        fi
+    if [ -x "`which yum 2>/dev/null`" ]; then
+        yum remove -y elasticsearch > /dev/null 2>&1 || true
+    fi
 
-        if [ -x "`which apt-get 2>/dev/null`" ]; then
-            apt-get --quiet --yes purge elasticsearch > /dev/null 2>&1 || true
-        fi
+    # Removes DEB package
+    if is_dpkg; then
+        dpkg --purge elasticsearch > /dev/null 2>&1 || true
+    fi
 
-        # Removes user & group
-        userdel elasticsearch > /dev/null 2>&1 || true
-        groupdel elasticsearch > /dev/null 2>&1 || true
+    if [ -x "`which apt-get 2>/dev/null`" ]; then
+        apt-get --quiet --yes purge elasticsearch > /dev/null 2>&1 || true
+    fi
 
+    # Removes user & group
+    userdel elasticsearch > /dev/null 2>&1 || true
+    groupdel elasticsearch > /dev/null 2>&1 || true
 
-        # Removes all files
-        for d in "${ELASTICSEARCH_TEST_FILES[@]}"; do
-            if [ -e "$d" ]; then
-                rm -rf "$d"
-            fi
-        done
-    fi
 
-    # Checks that all files are deleted
+    # Removes all files
```

Comment:
The idea here was to fail and not start the test if removing a package left files around... But I think this is also tested in rpm/deb bats tests so it seems better to ensure everything is removed here.